OWML.ModHelper.Input/ModInputCombination.cs

```diff
@@ -0,0 +1,71 @@
+﻿using System.Collections.Generic;
+using OWML.Common;
+using UnityEngine;
+
+namespace OWML.ModHelper.Input
+{
+    public class ModInputCombination : IModInputCombination
+    {
+        public float LastPressedMoment => _lastPressedMoment;
```

Comment:
`public float LastPressedMoment { get; private set; }`
then you can remove _lastPressedMoment

Reply:
done